src/App/Components/Elements/FullPageModal/full-page-modal.jsx

```diff
@@ -65,16 +26,34 @@ class FullPageModal extends React.Component {
             title,
         } = this.props;
         return (
-            <PoseGroup>
-                {(is_visible && !is_loading) && [
-                    <ModalBackground
-                        className='full-page-modal__background'
-                        key='full-page-modal__background'
-                    />,
-                    <ModalWrapper
-                        className='full-page-modal__wrapper'
-                        key='full-page-modal__wrapper'
-                    >
+            <React.Fragment>
+                <CSSTransition
```

Comment:
Is there a plan to replace the other `<Pose>` transitions with `<CSSTransition>`? I think we don't want to have two different ways of handling transitions.

Reply:
Yeah, I'll probably revert back all to `CSSTransition` since `react-pose` seems to be not as efficient in managing the transitions between lifecycle states when used in tandem with `mobx`.